Apps/Admin/Client/Store/UserFeedback/UserFeedbackReducers.cs

```diff
@@ -116,24 +117,24 @@ public static UserFeedbackState ReduceDisassociateTagAction(UserFeedbackState st
     /// <param name="action">The load success action.</param>
     /// <returns>The new state.</returns>
     [ReducerMethod]
-    public static UserFeedbackState ReduceDisassociateTagSuccessAction(UserFeedbackState state, UserFeedbackActions.DisassociateTagSuccessAction action)
+    public static UserFeedbackState ReduceDissociateTagSuccessAction(UserFeedbackState state, UserFeedbackActions.DissociateTagSuccessAction action)
     {
-        IImmutableDictionary<Guid, UserFeedbackTagView> data = state.FeedbackTagData ?? new Dictionary<Guid, UserFeedbackTagView>().ToImmutableDictionary();
+        IImmutableDictionary<Guid, UserFeedbackView> data = state.FeedbackData ?? new Dictionary<Guid, UserFeedbackView>().ToImmutableDictionary();
 
-        if (action.Result.ResourcePayload)
+        if (action.Result.ResourcePayload && data.TryGetValue(action.FeedbackId, out UserFeedbackView? feedback))
         {
-            data = data.Remove(action.Id);
+            feedback?.Tags.Remove(action.FeedbackTag);
```

Comment:
This likely won't work since `Remove()` will try to find a match based on reference equality.  Instead you could try:

```
UserFeedbackTagView? tag = feedback.Tags.SingleOrDefault(t => t.Tag.Id == action.FeedbackTag.Id);
if (tag != null)
{
    feedback.Tags.Remove(tag);
}
```